### ISSUE TITLE
Funds Not Unstake on Recovery from Recovery Address

### ISSUE THREAD

**Issue by zeshann110** [polymorpher/one-wallet#334]
I have staked some funds but when I test to recover the wallet by sending 1$ One from the Recovery address all funds receives but the stake funds were not received.

**Comment by polymorpher** [comment]
This will be addressed in a few weeks

**Comment by wmcbean** [comment]
Same issue I am facing as well, any fix for this?

**Comment by polymorpher** [comment]
Yes, there is a way to use your recovery wallet to control the expired wallet to do anything, though this feature is never exposed at the UI. I can implement a connector later, e.g. If your recovery wallet is MetaMask, the connector will let you enter the expired wallet, and use MetaMask to unstake and later send funds back to your recovery wallet.

**Comment by wmcbean** [comment]
That would be great, thank you. Let me know what I need to do on my side?

**Comment by polymorpher** [comment]
How much was locked in the address? Can you send me the address again or post it here?

**Comment by wmcbean** [comment]
Hi. This is the wallet address  one1zmfwlje82smun9a2505smkvmlr49yk9celjyas,
at the time over 10,000 one was staking

**Comment by polymorpher** [comment]
Will follow up when that feature is released. You will need to connect with your recovery wallet https://explorer.harmony.one/address/0xBa1649faF001cB4aBFe9328A060f6eC3Dc39a7f1  then control the old expired wallet to unstake and transfer the money back

**Comment by wmcbean** [comment]
Ok, I will wait. Thanks
